public/app/features/alerting/unified/components/notification-policies/Policy.test.tsx

```diff
@@ -10,18 +10,18 @@ import {
   AlertmanagerGroup,
   MatcherOperator,
   ObjectMatcher,
-  RouteWithID,
+  RouteWithID
 } from 'app/plugins/datasource/alertmanager/types';
 import { ReceiversState } from 'app/types/alerting';
 
 import { mockAlertGroup, mockAlertmanagerAlert, mockReceiversState } from '../../mocks';
 import { GRAFANA_RULES_SOURCE_NAME } from '../../utils/datasource';
 
 import { Policy } from './Policy';
+import { shouldShowExportOption } from './shouldShowExportOption';
 
-beforeAll(() => {
```

Comment:
as we should use the return of userEvent.setup() for the events in each test.

Reply:
Nice!